test/calculatorTest.js

```diff
@@ -0,0 +1,28 @@
+const chai = require('chai');
+const { assert } = require('chai');
+const calculator = require('../calculator')
+
+
+describe('Test addition', ()=> {
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bakersen/code-quality-badges/pullRequest?prid=8111319)
```suggestion
describe('Test addition', () => {
```